src/components/navigation-tray/nav-tray.jsx

```diff
@@ -0,0 +1,71 @@
+/// This component is a navigation tray
+/// for mobile version of the app
+
+import { NavLink } from "react-router-dom";
+import notificationSign from "../../assets/notification_sign.svg";
+import styled from "styled-components";
+
+/// Here is a styles for every link:
+
+const StyledNavLink = styled(NavLink)`
+  width: 60px;
+  height: 60px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: center;
+  position: relative;
+  background-image: url("${({ inactive }) => inactive}");
+  &:active {
+    background-color: rgba(255, 255, 255, 0.3);
+  }
+  &.active {
+    background-color: rgba(255, 255, 255, 0.3);
+    background-image: url("${({ active }) => active}");
+  }
+`;
+
+const StyledNavBar = styled.nav`
+  width: 215px;
```

Comment:
See if you can get this to work without using a fixed width; if they ever want to add a link to it, we would want it to expand freely to accommodate it